CHANGELOG.rst

```diff
@@ -1,5 +1,60 @@
 .. _changelog:
 
+0.6.3
+-----
+
+New feature
+~~~~~~~~~~~~
+
+⭐️ Project details now include a listing of the commit history
+
+Notable improvements
+~~~~~~~~~~~~~~~~~~~~
+
+🚄 Environments: auto-saved branches are filtered per username
+
+🚄 Improve markdown rendering and code highlighting
+
+🚄 Editing markdown files is easier as ``ckeditor`` is partially integrated inside Renku
+
+Bug fixes
+~~~~~~~~~
+
+* Dataset contains all folders from unzipped file
+* Failing to retrieve metadata for one dataset does not cause the others to fail
+* Improved UX for when datasets take too long
+* Datasets: no failure when adding ignored files
+
+Miscellaneous
+^^^^^^^^^^^^^
+
+- Default R templates now use the latest R (4.0.0); `bioc 3_11 <https://www.bioconductor.org/news/bioc_3_11_release>`__ also now available
+- Docker images in project templates use ``renku`` `0.10.4 <https://github.com/SwissDataScienceCenter/renku-python/releases/tag/0.10.4>`__
```

Comment:
link url should be: https://github.com/SwissDataScienceCenter/renku-python/releases/tag/v0.10.4

Reply:
yep, nice catch